Components/badge.css

```diff
@@ -0,0 +1,78 @@
+.badge{
+    padding: 0.2rem 0.65rem;
```

Comment:
Try to use globally defined variables for properties like margin, padding and gaps

Reply:
Sure, i will make changes